packages/contracts/contracts/QuadraticVoteCollectModule.sol

```diff
@@ -147,67 +107,48 @@ contract QuadraticVoteCollectModule is FeeModuleBase, FollowValidationModuleBase
         uint256 pubId,
         bytes calldata data
     ) internal {
-        /// decode  data
         (address _currency, uint256 _amount) = abi.decode(data, (address, uint256));
-        {
-            uint256 roundStartTime = IRoundImplementation(
-                _dataByPublicationByProfile[profileId][pubId].grantsRoundAddress
-            ).roundStartTime();
-            uint256 roundEndTime = IRoundImplementation(
-                _dataByPublicationByProfile[profileId][pubId].grantsRoundAddress
-            ).roundEndTime();
-            require(block.timestamp > roundStartTime && block.timestamp < roundEndTime, "Round is not in session");
-        }
-        
-        uint256 amount = _dataByPublicationByProfile[profileId][pubId].amount;
-        address currency = _dataByPublicationByProfile[profileId][pubId].currency;
-        uint256 referralFee = _dataByPublicationByProfile[profileId][pubId].referralFee;
-        //validate data
-        _validateDataIsExpected(data, currency, amount);
 
-        //transfer erc20 from collector to this contract
-        IERC20(_currency).safeTransferFrom(collector, address(this), _amount);
+        uint256 referralFee = _dataByPublicationByProfile[profileId][pubId].referralFee;
+        _validateDataIsExpected(data, _currency, _amount);
```

Comment:
if you're just passing in the currency and amount from the decoded data then this check doesn't do anything.  
[look here for reference](https://github.com/lens-protocol/core/blob/9f3b62aab1a4983d9bc11ee4f5c2892e8f1f148a/contracts/core/modules/FeeModuleBase.sol#L35)
[and here](https://github.com/lens-protocol/core/blob/main/contracts/core/modules/collect/FeeCollectModule.sol#L155)

Reply:
Well we don't know the amount and arguably you could say the same for the currency except that they need to be accepted as a 'voting currency' in the Grants round